### ISSUE TITLE
Content List style question and error

### ISSUE THREAD

**Issue by wturnbull** [CuBoulder/tiamat-theme#1169]
Curious if the styles are set for the Content List. The following examples are set as sidebar in D7 and D10:

D7: https://www.colorado.edu/research/research-report/research-creative-work-2022-23
<img width="721" alt="Screenshot 2024-08-01 at 3 53 37 PM" src="https://github.com/user-attachments/assets/4484ee3a-bc1d-48f2-a993-9579fb60fbab">

D10 (note I couldn't screenshot the entire block but it's on this page): https://live-ucbdev-homepage.pantheonsite.io/research/report/2022-23
![Screenshot 2024-08-01 at 3 54 14 PM](https://github.com/user-attachments/assets/6b4783ed-63f9-4b19-b8ee-cb3d8f9f7380)

The images are a lot bigger on the D10 site which makes it not a good option to do just a title and image. Also seeing a formatting issue with the last article listed on D10 with that one image aligned to the left and under the title:

![Screenshot 2024-08-01 at 3 56 33 PM](https://github.com/user-attachments/assets/f853ef7a-6e25-4107-a9c3-7684a30a9c02)

**Comment by patrickbrown-io** [comment]
Checking with Kevin for guidance

**Comment by kevincrafts** [comment]
The thumbnail + title display should match the D7 version with the slightly smaller square thumbnail floated to the left.

**Comment by wturnbull** [comment]
Hi all! I'm still seeing issues with the content lists and formatting. See bottom area of these pages:

https://live-ucbdev-homepage.pantheonsite.io/research/report/2022-23
https://live-ucbdev-homepage.pantheonsite.io/research/report/2021-22
https://live-ucbdev-homepage.pantheonsite.io/research/report/2020-21
https://live-ucbdev-homepage.pantheonsite.io/research/report/2019-20

**Comment by jbertrand** [comment]
@jcsparks Do we have a status of this one. It has been on the list for awhile and we had to re-open it two weeks. The issues are still there.

**Comment by jcsparks** [comment]
@jbertrand  - the original ticket was about the icon size.  I think that's been resolved.  Looking at the display of the content, I think the issue we have is the formatting of the text next to the icon.  

Currently it's displaying like this :
![Screenshot 2024-11-18 at 11 49 17 AM](https://github.com/user-attachments/assets/87039655-84db-4737-a29d-686cb1be14a5)


I think you want it to be like this : 

![Screenshot 2024-11-18 at 11 48 41 AM](https://github.com/user-attachments/assets/cc5e1e02-9052-4948-b915-1051f87bf1e4)

Assuming that's the fix we need to make, I'll have the Devs make that change.

**Comment by wturnbull** [comment]
Yes, it's the display that is the issue - sometimes it has the thumbnail to the left of the title (which is accurate) and sometimes the thumbnail appears above the title.

**Comment by jcsparks** [comment]
@patrickbrown-io - we just need to change the width on the content list text container, it's currently set to `fit-content`, changing to this should do the trick.  

`.ucb-content-list-sidebar .ucb-content-list-text-container {
    width: min-content;
    flex-grow: 1;
}`
